src/ClientWrapper.php

```diff
@@ -4,28 +4,41 @@
 
 use Exceptions\Http\Client\MethodNotAllowedException;
 
-class AbstractClient
+class ClientWrapper
 {
     /**
      * @var HttpClient
      */
     private $client;
     private $logger;
+    private $header;
 
-    public function __construct($baseUrl,$logger=null)
+    public function __construct($baseUrl,$token, $logger=null)
```

Comment:
I think $token should be managed in ClientWrapper instead of here, right? 
@liang611zh is that what you mean?